### ISSUE TITLE
Already watched movie on Trakt won't scrobble a second time

### ISSUE THREAD

**Issue by zeroquinc** [Taxel/PlexTraktSync#998]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists

### The problem

Whenever I scrobble a movie that got scrobbled to Trakt before, it won't scrobble a second time.

### Error trace / logs

```text
INFO     on_play: <Movie:1188:John-Wick:-Chapter-2>: 99.531488% Watched: True,  
         LastViewed: 2022-08-04 17:42:50                                        
INFO     on_play: <Movie:1188:John-Wick:-Chapter-2>: 99.667684% Watched: True,  
         LastViewed: 2022-08-04 17:42:50                                        
INFO     on_play: <Movie:1188:John-Wick:-Chapter-2>: 99.803879% Watched: True,  
         LastViewed: 2022-08-04 17:42:50                                        
INFO     on_play: <Movie:1188:John-Wick:-Chapter-2>: 99.940074% Watched: True,  
         LastViewed: 2022-08-04 17:42:50                                        
INFO     on_play: <Movie:1188:John-Wick:-Chapter-2>: 99.994552% Watched: True,  
         LastViewed: 2022-08-04 17:42:50                                        
INFO     on_activity: <tmdb:324552:Movie:1188:John-Wick:-Chapter-2>: Collected: 
         True, Watched: [Plex: True, Trakt: True]
```

I use this config.

```text
sync:
  plex_to_trakt:
    collection: true
    ratings: true
    watched_status: false
  trakt_to_plex:
    liked_lists: false
    ratings: false
    watched_status: true
    watchlist: false

watch:
  add_collection: true
  remove_collection: false
  scrobble_threshold: 100
  username_filter: true
```

### Expected behavior

I expect that it still scrobbles even when it's already scrobbled before on Trakt.

### Steps to reproduce the behavior

1. Run the watch command with `scrobble_threshold` on `100`
2. Pick a movie that already got scrobbled to Trakt once and watch it or skip to the end.
3. Notice how it doesn't get scrobbled to Trakt.

### Install method

docker-compose

### Version

0.20.9

### Python Version

3.10.5

### Operating System and Version

Ubuntu 22.04

**Comment by twolaw** [comment]
Why did you set the `scrobble_threshold `to `100` ? (default is `80`)
Can you explain how you think it is supposed to work.

**Comment by zeroquinc** [comment]
> Why did you set the `scrobble_threshold `to `100` ? (default is `80`) Can you explain how you think it is supposed to work.

I set the `scrobble_threshold` to `100` because if I didn't set it to `100` I got a bug where all my plays got scrobbled twice.

**Comment by twolaw** [comment]
> I set the `scrobble_threshold` to `100` because if I didn't set it to `100` I got a bug where all my plays got scrobbled twice.

Ok, this bug has been fixed in version `0.20.7` and the workaround is not needed anymore.
You can set `scrobble_threshold` back to `80` it will solve your problem.

**Comment by zeroquinc** [comment]
It is indeed solved!
